artiq/master/worker_impl.py

```diff
@@ -286,8 +287,8 @@ def main():
                     dataset_mgr.write_hdf5(f)
                     f["artiq_version"] = artiq_version
                     f["rid"] = rid
-                    f["start_time"] = int(time.mktime(start_time))
-                    f["run_time"] = int(time.mktime(run_time))
+                    f["start_time"] = int(start_time)
+                    f["run_time"] = int(run_time)
```

Comment:
Let's make these doubles while we are at it. I see no reason to truncate it here.